apps/builder/app/dashboard/projects/project-card.tsx

```diff
@@ -206,8 +217,18 @@ export const ProjectCard = ({
   return (
     <Card hidden={isHidden} tabIndex={0} onKeyDown={handleKeyDown}>
       <CardContent
-        css={{ background: theme.colors.brandBackgroundProjectCardBack }}
+        css={{
+          background: theme.colors.brandBackgroundProjectCardBack,
+          position: "relative",
+          [`&:hover ${EmptyImage}`]: {
```

Comment:
we don't use that any more, we use a css variable and we have a naming guideline for css variables (just in case https://github.com/webstudio-is/webstudio-rfcs/blob/main/text/0003-css-vars.md)